### PR TITLE
Add release info for lm-agent

### PR DESCRIPTION
**Pull request by julianaklulo**
#### What
Add release information for License Manager Agent.
Also add version 2.1.0.dev3 to CHANGELOG.

#### Why
We need to document the release process for the agent.

`Task`: https://app.clickup.com/t/18022949/LM-150

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,6 +194,16 @@ docker-compose up --build
 uvicorn licensemanager2.agent.main:app --port 8010
 ```
 
+## Release (agent)
+To make a new release of the License Manager Agent:
+
+1. Bump the version in the `pyproject.toml` file
+2. Update the CHANGELOG file, moving the changes under the Unreleased section to the new version section.
```
Comment (heitorPB) [inline README.md]:
```suggestion
2. Update the `CHANGELOG` file, moving the changes under the `Unreleased` section to the new version section. Always keep a `Unreleased` section at the top.
3. Create a commit with title `Release x.y.z`.
```

Comment (julianaklulo) [reply README.md]: Thanks, I added it.

**Review by heitorPB — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -194,6 +194,17 @@ docker-compose up --build
...
+To make a new release of the License Manager Agent:
+
+1. Bump the version in the `pyproject.toml` file.
+2. Update the `CHANGELOG` file, moving the changes under the Unreleased section to the new version section. Always keep an `Unreleased` section at the top.
+3. Create a new commit with the title `Release x.y.z`.
+3. Use Poetry to publish the new version to Pypicloud (the credentials can be found on `1Password`)
+```bash
+poetry publish --build --repository pypicloud --username <username> --password <password>
+```
+
```
Comment (jamesbeedy) [inline README.md:207]: 👍

**Review by jamesbeedy — APPROVED** [review]
👍

**Pull request merged**